packages/components/psammead-copyright/README.md

```diff
@@ -16,13 +16,16 @@ Displays a source attribution in block capitals in the bottom-right of the paren
 
 ## Usage
 
-Commonly used alongside [`psammead-figure`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-figure) and [`psammead-image`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image).
+Commonly used alongside [`psammead-figure`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-figure) and [`psammead-image`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image). Can also pass in [`psammead-visually-hidden-text`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-visually-hidden-text) in order to announce the component.
 
 ```jsx
 const WrapperComponent = ({ alt, src, width }) => (
   <Figure>
     <Image alt={alt} src={src} width={width} />
-    <Copyright>Getty Images</Copyright>
+    <Copyright>
+      <VisuallyHiddenText>Copyright </VisuallyHiddenText>
```

Comment:
UX Designers have recommended using `Image source, ` as the prefixed string that's used offscreen, so it would be good to have our reflected in the example usage. 
```suggestion
      <VisuallyHiddenText>Image source, </VisuallyHiddenText>
```